src/app/navbar/navbar.component.ts

```diff
@@ -8,13 +9,25 @@ import { Component, OnInit } from '@angular/core';
 })
 export class NavbarComponent /*implements OnInit*/ {
 
-  constructor() { }
+  logoutSuccess = false;
+  status: Number;
+
+  constructor(private userLogoutService: UserLogoutService, private router: Router) { }
 
   clickLogin() {
     //showLoginPage();
   }
 
   /*ngOnInit(): void {
   }*/
+  handleLogout() {
+    this.userLogoutService.logoutUser(JSON.parse(localStorage.getItem('UserID'))).subscribe((result) => {
+      console.log(JSON.parse(localStorage.getItem('UserID')))
+      localStorage.removeItem('UserID');
+      this.logoutSuccess = true;
+      this.status = +result;
+      this.router.navigate(['/login'])
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7552947)
```suggestion
      this.router.navigate(["/login"])
```